gprofiler/metadata/application_metadata.py

```diff
@@ -55,4 +55,7 @@ def get_metadata(self, process: Process) -> Optional[Dict]:
         return metadata
 
     def make_application_metadata(self, process: Process) -> Dict[str, Any]:
-        return {"exe": process.exe(), "execfn": read_process_execfn(process)}
+        # Disabled for lack of Windows support
+        #return {"exe": process.exe(), "execfn": read_process_execfn(process)}
+        return {"exe": process.exe(), "execfn": None}
```

Comment:
Add a helper function `is_windows()` that checks the platform, then pass `None` if Windows and otherwise call `read_process_execfn` as the code does now